src/components/main-view/main-view.jsx

```diff
@@ -1,67 +1,29 @@
-      imagePath: "https://upload.wikimedia.org/wikipedia/en/1/1d/The_Shining_%281980%29_U.K._release_poster_-_The_tide_of_terror_that_swept_America_IS_HERE.jpg",
-      featured: true,
-    },
-    {
-      genre: {
-        name: "Sci-Fi",
-        description:
-          "Science fiction (sometimes shortened to SF or sci-fi) is a genre of speculative fiction, which typically deals with imaginative and futuristic concepts such as advanced science and technology, space exploration, time travel, parallel universes, and extraterrestrial life. It is related to fantasy, horror, and superhero fiction and contains many subgenres. Its exact definition has long been disputed among authors, critics, scholars, and readers.",
-      },
-      director: {
-        name: "Stanley Kubrick",
-        bio: "Director Stanley Kubrick was one of the most consistently fascinating filmmakers in the latter half of the 20th century. Just as his singularly brilliant visual style won him great acclaim, his unconventional sense of narrative and seeming lack of overt emotionalism often elicited critical puzzlement",
-        birth: "1928",
-        death: "1999",
-      },
-      id: 3,
-      title: "2001: A Space Odyssey",
-      description:
-        "An imposing black structure provides a connection between the past and the future in this enigmatic adaptation of a short story by revered sci-fi author Arthur C. Clarke. When Dr. Dave Bowman (Keir Dullea) and other astronauts are sent on a mysterious mission, their ship's computer system, HAL, begins to display increasingly strange behavior, leading up to a tense showdown between man and machine that results in a mind-bending trek through space and time.",
-      imagePath: "https://upload.wikimedia.org/wikipedia/en/1/11/2001_A_Space_Odyssey_%281968%29.png",
-      featured: true,
-    },
-  ])
+  const [movies, setMovies] = useState([]);
+
+  useEffect(() => {
+    fetch("https://my-movie-app-ab91e4bb4611.herokuapp.com/movies")
+      .then((response) => response.json())
+      .then((data) => {
+        console.log("data check:", data)
+        const moviesFromApi = data.map((movie) => {
+          return {
+            id: movie._id,
+            title: movie.Title,
+            description: movie.Description,
+            genreName: movie.Genre.Name,
+            genreDescription: movie.Genre.Description,
+            directorName: movie.Director.Name,
+            directorBio: movie.Director.Bio,
+          };
+        });
+        setMovies(moviesFromApi);
+      });
+  }, []);
 
   const [selectedMovie, setSelectedMovie] = useState(null);
```

Comment:
Please move this line to just below line 6, it's best practice to group similar blocks of code. ie useState hooks..